autotest/gdrivers/zarr_driver.py

```diff
@@ -1011,15 +1023,16 @@ def test_zarr_read_v3(use_get_names):
     assert subgroup.OpenMDArray("not_existing") is None
 
 
-@pytest.mark.parametrize("endianness", ["le", "be"])
-def test_zarr_read_half_float(endianness):
-
-    filename = "data/zarr/f2_" + endianness + ".zarr"
-    ds = gdal.OpenEx(filename, gdal.OF_MULTIDIM_RASTER)
-    assert ds is not None
-    rg = ds.GetRootGroup()
-    ar = rg.OpenMDArray(rg.GetMDArrayNames()[0])
-    assert ar.Read() == array.array("f", [1.5, float("nan")])
+# @pytest.mark.parametrize("endianness", ["le", "be"])
```

Comment:
Why is this commented out?

Reply:
Thanks for noticing. I enabled the test again.

The test results didn't change because float16 values are automatically converted to float32 if using the Python API since SWIG cannot handle float16.